src/components/App.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+
+export interface HelloProps { greet: string; company: string }
```

Comment:
Semicolon should be there. We are following these rules so. linting will give you an error for this but it is still not configured so I am adding a comment.

Reply:
Yes !! I will be configuring it in another branch!!